src/controller/CHIPDeviceController.cpp

```diff
@@ -3041,7 +3041,7 @@ void DeviceCommissioner::PerformCommissioningStep(DeviceProxy * proxy, Commissio
 
         CHIP_ERROR err = CHIP_NO_ERROR;
         ChipLogProgress(Controller, "SendCommand kWiFiNetworkEnable, supportsConcurrentConnection=%d",
-                        params.GetSupportsConcurrentConnection().Value());
+                        params.GetSupportsConcurrentConnection().ValueOr(false));
```

Comment:
This makes the log not clear about if `false` or `optional::Missing` is there. We should probably use `%s` and the logic of `params.GetSupportsConcurrentConnection().HasValue() ? (params.GetSupportsConcurrentConnection().Value() ? "true" : "false") : "missing"` (or better place this in some getter function) .... very verbose but does not hide away things.

Reply:
I understand. I modified.